lib/govuk_document_types/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukDocumentTypes
-  VERSION = "0.2.0".freeze
+  VERSION = "0.2.1".freeze
```

Comment:
As @vanitabarrett mentioned elsewhere, this should be in a separate PR along with an update to the change note.

Also, I think this counts as minor change, rather than a patch, so should be updated to `0.3.0`.